### DISCUSSION TITLE
Download Lyrics

### DISCUSSION THREAD

**Discussion by Ibrahim-Rashwan** [Shabinder/SpotiFlyer#1188] [Ideas]
Download the lyrics for the song automatically

**Comment by pcm8081** [comment]
you can copy paste. or download lrc  from 
https://www.megalobiz.com
some android player which work perfectly  to show auto scroll lyrics  >from  embedded lyrics
minilyrics, gonemad (both work perfectly ), otto, 🌷retro music🌷,musicolet,aimp,pulsar,....
in pc-for mp3/m4a....-minilyrics+(aimp+foobar+wmp+musicbee...vlc)
for m4a-pot player

**Reply by kcryptonian** [reply]
It would be good if the spotiflyer could download the embedded lyrics alojg with the music file.
